### DISCUSSION TITLE
how to send sms from draft?

### DISCUSSION THREAD

**Discussion by krzysiekhelp** [Salamek/huawei-lte-api#178] [General]
is it possible to send via api sms from the draft directory?
sometimes sms are not sent and saved as draft

**Comment by iursevla** [comment]
@krzysiekhelp did you solve it?

**Comment by krzysiekhelp** [comment]
regularly deletes sent items.  and then there are no unsent messages.

niedz., 24 mar 2024, 11:32 użytkownik Rui Alves ***@***.***>
napisał:
